### PR TITLE
chore: sonar output is missing

### PR DESCRIPTION
**Pull request by mswertz**
We saw notices in CDCD log on step **build, test, push docker**

Both PRs below have same related log ... guess not generating a report could make next sonar step fail.

> Task :jacocoTestReport SKIPPED

## This PR #695

> The property 'sonar.login' is deprecated and will be removed in the future. Please use the 'sonar.token' property instead when passing a token.

> No coverage report can be found with sonar.coverage.jacoco.xmlReportPaths='/root/repo/build/reports/jacoco/jacocoMergedReport/jacocoMergedReport.xml'. Using default locations: target/site/jacoco/jacoco.xml,target/site/jacoco-it/jacoco.xml,build/reports/jacoco/test/jacocoTestReport.xml

```log
> Task :r:jacocoTestReport
> Task :bootStartScripts
> Task :jacocoTestReport SKIPPED <=========
> Task :startScripts
> Task :jacocoMergedReport

> Task :sonar
The property 'sonar.login' is deprecated and will be removed in the future. Please use the 'sonar.token' property instead when passing a token.
No coverage report can be found with sonar.coverage.jacoco.xmlReportPaths='/root/repo/build/reports/jacoco/jacocoMergedReport/jacocoMergedReport.xml'. Using default locations: target/site/jacoco/jacoco.xml,target/site/jacoco-it/jacoco.xml,build/reports/jacoco/test/jacocoTestReport.xml
No coverage report can be found with sonar.coverage.jacoco.xmlReportPaths='/root/repo/build/reports/jacoco/jacocoMergedReport/jacocoMergedReport.xml'. Using default locations: target/site/jacoco/jacoco.xml,target/site/jacoco-it/jacoco.xml,build/reports/jacoco/test/jacocoTestReport.xml
No coverage report can be found with sonar.coverage.jacoco.xmlReportPaths='/root/repo/build/reports/jacoco/jacocoMergedReport/jacocoMergedReport.xml'. Using default locations: target/site/jacoco/jacoco.xml,target/site/jacoco-it/jacoco.xml,build/reports/jacoco/test/jacocoTestReport.xml
No coverage report can be found with sonar.coverage.jacoco.xmlReportPaths='/root/repo/build/reports/jacoco/jacocoMergedReport/jacocoMergedReport.xml'. Using default locations: target/site/jacoco/jacoco.xml,target/site/jacoco-it/jacoco.xml,build/reports/jacoco/test/jacocoTestReport.xml

> Task :dockerClean
> Task :dockerPrepare
```

## Other PR ie #698

```
> Task :r:jacocoTestReport
> Task :bootStartScripts
> Task :jacocoTestReport SKIPPED <==========
> Task :startScripts
> Task :jacocoMergedReport

> Task :sonar
The property 'sonar.login' is deprecated and will be removed in the future. Please use the 'sonar.token' property instead when passing a token.
No coverage report can be found with sonar.coverage.jacoco.xmlReportPaths='/root/repo/build/reports/jacoco/jacocoMergedReport/jacocoMergedReport.xml'. Using default locations: target/site/jacoco/jacoco.xml,target/site/jacoco-it/jacoco.xml,build/reports/jacoco/test/jacocoTestReport.xml
No coverage report can be found with sonar.coverage.jacoco.xmlReportPaths='/root/repo/build/reports/jacoco/jacocoMergedReport/jacocoMergedReport.xml'. Using default locations: target/site/jacoco/jacoco.xml,target/site/jacoco-it/jacoco.xml,build/reports/jacoco/test/jacocoTestReport.xml
No coverage report can be found with sonar.coverage.jacoco.xmlReportPaths='/root/repo/build/reports/jacoco/jacocoMergedReport/jacocoMergedReport.xml'. Using default locations: target/site/jacoco/jacoco.xml,target/site/jacoco-it/jacoco.xml,build/reports/jacoco/test/jacocoTestReport.xml
No coverage report can be found with sonar.coverage.jacoco.xmlReportPaths='/root/repo/build/reports/jacoco/jacocoMergedReport/jacocoMergedReport.xml'. Using default locations: target/site/jacoco/jacoco.xml,target/site/jacoco-it/jacoco.xml,build/reports/jacoco/test/jacocoTestReport.xml

> Task :dockerClean
> Task :dockerPrepare
```

### CODE REVIEW

**Comment by clemens-tolboom** [comment]
Not sure whether this is needed as https://sonarcloud.io/project/pull_requests_list?id=org.molgenis%3Aarmadillo-service seems alive

File: build.gradle [hunk 1]
```diff
@@ -155,6 +155,8 @@ task jacocoMergedReport(type: JacocoReport) {
     executionData.setFrom project.fileTree(dir: '.', include: '**/build/jacoco/test.exec')
     reports {
         xml.required
+        csv.required = false
+        html.required = false
```
Comment (clemens-tolboom) [inline build.gradle:159]: This is weird ... `xml.required` versus `css.required = false`

**Comment by clemens-tolboom** [comment]
```log
> Task :r:jacocoTestReport
> Task :bootStartScripts
> Task :jacocoTestReport SKIPPED
> Task :startScripts
> Task :jacocoMergedReport
```

**Comment by clemens-tolboom** [comment]
Running locally

```bash
./gradlew build jacocoMergedReport
```
gives

> Task :jacocoMergedReport
Execution optimizations have been disabled for task ':jacocoMergedReport' to ensure correctness due to the following reasons:
>  - Gradle detected a problem with the following location: '/Users/clemens/Documents/GitHub/molgenis-service-armadillo'. Reason: Task ':jacocoMergedReport' uses this output of task ':armadillo:spotlessJava' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. Please refer to https://docs.gradle.org/7.6.4/userguide/validation_problems.html#implicit_dependency for more details about this problem.
>  - Gradle detected a problem with the following location: '/Users/clemens/Documents/GitHub/molgenis-service-armadillo'. Reason: Task ':jacocoMergedReport' uses this output of task ':r:spotlessJava' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. Please refer to https://docs.gradle.org/7.6.4/userguide/validation_problems.html#implicit_dependency for more details about this problem.
```

**Comment by clemens-tolboom** [comment]
Still skipping

```
RExecutorServiceImplTest > testLoadResource() PASSED

> Task :r:spotlessJava
> Task :r:spotlessJavaApply
> Task :r:spotlessApply
> Task :r:jacocoTestReport
> Task :bootStartScripts
> Task :jacocoTestReport SKIPPED
> Task :startScripts
> Task :jacocoMergedReport

> Task :sonar
```

File: r/build.gradle [hunk 2]
```diff
@@ -72,8 +64,17 @@ test {
     finalizedBy jacocoTestReport
 }
 
+jacocoTestReport {
+    reports {
+        xml.required
+    }
```
Comment (clemens-tolboom) [inline r/build.gradle:70]: This is missing boolean

**Comment by mswertz** [comment]
sorry i broke it. Probably better to start with empty PR again.

**Comment by clemens-tolboom** [comment]
build gradle files is now #711

**Comment by mswertz** [comment]
Normally sonar should lead to a 'check'

**Pull request closed without merging**